src/react-native/react-native-wizard.ts

```diff
@@ -0,0 +1,256 @@
+
+  const js = fs.readFileSync(jsPath, 'utf-8');
+  if (
+    doesJsCodeIncludeSdkSentryImport(js, { sdkPackageName: RN_SDK_PACKAGE })
+  ) {
+    clack.log.warn(`${chalk.bold(jsRelativePath)} already includes Sentry.`);
+    return;
+  }
+
+  const newContent = addSentryInitWithSdkImport(js, { dsn });
+
+  clack.log.success(
+    chalk.greenBright(
+      `Added ${chalk.bold('Sentry.init')} to ${chalk.bold(jsRelativePath)}.`,
+    ),
+  );
+
+  fs.writeFileSync(jsPath, newContent, 'utf-8');
+  clack.log.success(
+    chalk.greenBright(`${chalk.bold(jsRelativePath)} changes saved.`),
+  );
+}
+
+async function confirmFirstSentryException(project: SentryProjectData) {
+  const projectsIssuesUrl = `${project.organization.links.organizationUrl}/issues/?project=${project.id}`;
+
+  clack.note(`To make sure everything is set up correctly, put the following code snippet into your application.
+The snippet will create a button that, when tapped, sends a test event to Sentry.
+
+After that check your project issues:
+
+${chalk.cyan(projectsIssuesUrl)}
+
+<Button title='Try!' onPress={ () => { Sentry.captureException(new Error('First error')) }}/>`);
+
+  const firstErrorConfirmed = clack.confirm({
+    message: `Have you successfully sent a test event?`,
+  });
+
+  return firstErrorConfirmed;
+}
+
+async function patchXcodeFiles({ authToken }: { authToken: string }) {
+  await addSentryCliConfig(authToken, {
+    ...propertiesCliSetupConfig,
+    name: 'source maps and iOS debug files',
+    filename: 'ios/sentry.properties',
+  });
+
+  if (platform() === 'darwin') {
+    await podInstall('ios');
+  }
+
+  const xcodeProjectPath = getFirstMatchedPath(XCODE_PROJECT);
+  if (!xcodeProjectPath) {
+    clack.log.warn(
+      `Could not find Xcode project file using ${chalk.bold(XCODE_PROJECT)}.`,
+    );
+    return;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+  const xcodeProject = xcode.project(xcodeProjectPath);
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+  xcodeProject.parseSync();
+  const buildPhasesMap = getValidExistingBuildPhases(xcodeProject);
+
+  const bundlePhase = findBundlePhase(buildPhasesMap);
+  patchBundlePhase(bundlePhase);
+
+  const debugFilesUploadPhaseExists =
+    !!findDebugFilesUploadPhase(buildPhasesMap);
+  addDebugFilesUploadPhase(xcodeProject, { debugFilesUploadPhaseExists });
+
+  writeXcodeProject(xcodeProjectPath, xcodeProject);
+}
+
+async function patchAndroidFiles({ authToken }: { authToken: string }) {
+  await addSentryCliConfig(authToken, {
+    ...propertiesCliSetupConfig,
+    name: 'source maps and iOS debug files',
+    filename: 'android/sentry.properties',
+  });
+
+  const appBuildGradlePath = getFirstMatchedPath(APP_BUILD_GRADLE);
+  if (!appBuildGradlePath) {
+    clack.log.warn(
+      `Could not find Android ${chalk.bold(
```

Comment:
nit: Not just here but generally, I'd prefer filenames being `chalk.cyan` as defined [here](https://develop.sentry.dev/sdk/setup-wizards/#colors). Not a blocker though.